nuget/Vcpkg.Nuget.nuspec

```diff
@@ -5,7 +5,7 @@
     <id>Vcpkg.Nuget</id>
 
     <!-- The package version number that is used when resolving dependencies -->
-    <version>1.4.0</version>
+    <version>1.5.0</version>
```

Comment:
note to self: we'll want to publish this after taking this pr 😸

Reply:
Hopefully 😏